### PR TITLE
feat(data) : references range and more

### PR DESCRIPTION
**Pull request by rmelisson**
https://github.com/SocialGouv/code-du-travail-numerique/issues/2524

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "start": "node -r esm src/index.js",
     "lint": "eslint src --ext .js",
     "test": "jest",
-    "check-url": "node -r esm src/verifyUrl.js"
+    "check-url": "node -r esm src/verifyUrl.js",
```
Comment (lionelB) [inline package.json:21]: you can remove this one, it's was used to update url that have 301,  but it's no more used

File: package.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
...
+    "check-refs": "node -r esm src/checkRefs.js"
```
Comment (lionelB) [inline package.json:22]: I propose to add this script to the the [release job](https://github.com/SocialGouv/fiches-travail-data/blob/master/.github/workflows/release.yml#L33)

File: src/__tests__/referenceExtractor.test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import fs from "fs";
-import path from "path";
-import { classifyTokens, extractReferences } from "../referenceExtractor";
-import { resolveReferences } from "../referenceResolver";
+const fs = require("fs");
```
Comment (lionelB) [inline src/__tests__/referenceExtractor.test.js]: you can use import here too

File: package.json (hunk 2 above)
Comment (rmelisson) [reply package.json]: like [this](https://github.com/SocialGouv/fiches-travail-data/pull/5/files#diff-56c30d9224897b08c28075052d4595c5R34) ?

**Pull request merged**